scripts/beachball/utils.ts

```diff
@@ -33,12 +49,12 @@ function getVNextPackagePaths(allPackageInfo: AllPackageInfo) {
 }
 
 function getSharedPackagePaths(allPackageInfo: AllPackageInfo) {
+  // These packages depend on converged packages, but are private
+  // Can be included in the publish scope so that dependencies are bumped correctly.
+  const privateNonConverged = ['@fluentui/perf-test', '@fluentui/vr-tests'];
```

Comment:
**perf:** hoisted up so it's not re-created on every iteration

**question:** do we still need this ? we already separated v9 so I guess this can be removed completely ? cc @ling1726

Reply:
Yeah these are no longer needed because these projects no longer have v9 deps.

They were added because we needed to bump deps after a release or we would have a broken master branch thanks to syncpack

Reply:
awesome, I'll update the PR. ty!